test/bigtest/interactors/loan-history/loan-history-form.js

```diff
@@ -21,6 +21,7 @@ import CalloutInteractor from '@folio/stripes-components/lib/Callout/tests/inter
 
   duration = scoped('[data-test-period-duration]', TextFieldInteractor);
   interval = scoped('[data-test-period-interval]', SelectInteractor);
+  selector = scoped('[data-test-period-interval] select');
```

Comment:
`selector` is not quite clear enough what it is about. Please come up with a more clear name.